# python_ch03_problems.py